dipy/tracking/metrics.py

```diff
@@ -758,71 +758,6 @@ def _extrap(xyz, cumlen, distance):
     return Lambda*xyz[ind]+(1-Lambda)*xyz[ind-1]
 
 
-def downsample(xyz, n_pols=3):
```

Comment:
You can not delete this function, we need to respect a deprecation cycle for backward compatibility.
Here, how to do it:
- add the deprecation decorator like here: https://github.com/dipy/dipy/pull/1957/files#diff-096658fc22dd6806d39993cc4f1db5d8R30
- keep the function signature but call `set_number_points` underneath.